pkg/master/master.go

```diff
@@ -248,12 +248,6 @@ func (m *Master) InstallAPIs(c *Config) {
 			ParameterCodec:         api.ParameterCodec,
 			NegotiatedSerializer:   api.Codecs,
 		}
-		if autoscalingGroupVersion := (unversioned.GroupVersion{Group: "autoscaling", Version: "v1"}); registered.IsEnabledVersion(autoscalingGroupVersion) {
```

Comment:
Registration of deployments/scale and replicasets/scale shouldn't be gated by whether autoscaling/v1 is enabled or not (lines 721, 741)